src/connection-manager/wsHandling.ts

```diff
@@ -264,11 +264,15 @@ async function backtrackNetworkAmendmentStatus(
       index < currentLedger;
       index += 256
     ) {
+      // Use api V1 instead of the default (V2) since we don't have the correct
+      // LedgerResponseExpanded type for V2
+      // TODO: change to V2 once the issue is fixed.
```

Comment:
I recommend linking to the xrpl.js issue

Reply:
Hello, do you remember the details of the issue pertaining to API v2 ? We are able to use the API v2 in the `Ledger` request for the latest validated ledger (a few lines above). I'm trying to understand why that would be a problem for other ledger indices.

The [response format](https://xrpl.org/docs/references/http-websocket-apis/public-api-methods/ledger-methods/ledger#response-format) of the `Ledger` command appears to be compatible with the VHS code [here](https://github.com/ripple/validator-history-service/blob/51d5db952bb7a776651e7bb67a5fe416195b6d6e/src/connection-manager/wsHandling.ts#L189)

Reply:
Yes we dont need v1 now